bin/lib/versions.js

```diff
@@ -85,17 +85,59 @@ exports.get_apple_xcode_version = function() {
     child_process.exec('xcodebuild -version', function(error, stdout, stderr) {
         if (error) {
             d.reject(stderr);
+        } else {
+            var version = stdout.split('\n')[0].slice(6);
```

Comment:
Would be better to use a regexp here I think:
    var m = /Xcode (.*)/.exec(stdout);
    if (m) {
        d.resolve(m[1]);
    ...